dev/io.openliberty.persistence.3.0.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ConversionExceptionResource.nlsprops

```diff
@@ -27,7 +27,7 @@
 #   (single quote must be coded as one single quote '). 
 #
 # -------------------------------------------------------------------------------------------------
-3001=The object [{0}] object of the [{1}] class cannot be converted to [{2}].
+3001=The object [{0}], of class [{1}], could not be converted to [{2}].
```

Comment:
Change to `The [{0}] object of the [{1}] class cannot`

Reply:
I thought "could not" and "can not" were better than "couldn't" and "cannot" because they are more easily translated

Reply:
Please use "cannot" and do not use "could not" or "can not." Translation isn't a problem for cannot.